src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -2329,6 +2362,14 @@ ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
 
     uct_ib_mlx5_devx_check_dp_ordering(md, cap, cap_2, dev);
 
+    if (ucs_ternary_auto_value_is_yes_or_try(md_config->ddp_enable)) {
+        status = uct_ib_mlx5_devx_check_ddp(md, cap, cap_2, dev,
```

Comment:
if it's "try", then we should not `goto err_lru_cleanup;` on error.

Reply:
Its covered in: `uct_ib_mlx5_ddp_check_status`